cheatsheets/Cross_Site_Scripting_Prevention_Cheat_Sheet.md

```diff
@@ -180,9 +180,9 @@ Also, a `</script>` closing tag will close a script block even though it is insi
 
 #### RULE \#3.1 - HTML escape JSON values in an HTML context and read the data with JSON.parse
 
-In a Web 2.0 world, the need for having data dynamically generated by an application in a javascript context is common. One strategy is to make an AJAX call to get the values, but this isn't always performant. Often, an initial block of JSON is loaded into the page to act as a single place to store multiple values. This data is tricky, though not impossible, to escape correctly without breaking the format and content of the values.
+In a Web 2.0 world, the need for having data dynamically generated by an application in a JavaScript context is common. One strategy is to make an AJAX call to get the values, but this isn't always performant. Often, an initial block of JSON is loaded into the page to act as a single place to store multiple values. This data is tricky, though not impossible, to escape correctly without breaking the format and content of the values.
 
-**Ensure returned `Content-Type` header is application/json and not text/html.** This shall instruct the browser not misunderstand the context and execute injected script
+**Ensure returned `Content-Type` header is `application/json` and not `text/html`.** This shall instruct the browser not misunderstand the context and execute injected script
```

Comment:
I've enclosed it explicitly in the backquote because Textlint arranges it to `text/HTML`.